lib/msf/core/payload/windows/reverse_http.rb

```diff
@@ -218,6 +218,7 @@ def asm_reverse_http(opts={})
         0x04000000 | # INTERNET_NO_CACHE_WRITE
         0x00400000 | # INTERNET_FLAG_KEEP_CONNECTION
         0x00200000 | # INTERNET_FLAG_NO_AUTO_REDIRECT
+        0x00080000 | # INTERNET_FLAG_NO_COOKIE
```

Comment:
I believe it is supposed to be `INTERNET_FLAG_NO_COOKIES`, but the value is correct.